libs/docs/cdk/focusable-grid/examples/short-rows-example/short-rows-example.component.html

```diff
@@ -0,0 +1,13 @@
+<div fdkFocusableGrid shortRowFocus="first">
+    <div *ngFor="let list of [].constructor(5); let rowIndex = index" fdkFocusableList navigationDirection="horizontal">
```

Comment:
Confused at the purpose of `navigationDirection="horizontal"` here, isn't the direction both horizontal and vertical?